infra/label_evaluator_step_functions/lambdas/evaluate_labels.py

```
@@ -0,0 +1,380 @@
+            "root_run_id": receipt_trace.root_run_id,
+            "root_dotted_order": receipt_trace.root_dotted_order,
+        }
+
+        # End the receipt trace (don't close it yet - LLMReview will add to it)
+        # Note: We don't call end_receipt_trace here because LLMReview will continue it
+        # The trace will be ended after LLMReview completes
+
+        # Flush LangSmith traces
+        flush_langsmith_traces()
+
+        return output
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)